src/components/ArticleList.jsx

```diff
@@ -0,0 +1,44 @@
+import ArticleCard from './ArticleCard'
+import axios from 'axios';
+import {useEffect, useState} from 'react'
+
+const ArticleList = ()=>{
+
+    
+    const [articles,setArticles]=useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(()=>{
+        // axios
+        // .get('https://newsservicerg.onrender.com/api/articles')
+        // .then((response)=>{
+        //     setArticles(response.data.articles)
+        //     }
+        // )
+        fetch('https://newsservicerg.onrender.com/api/articles')
```

Comment:
It would be great if we could utilise axios here and extract api calls into their own file to keep our code tidy and DRY.